guidelines/sc/20/timing-adjustable.html

```diff
@@ -32,7 +32,8 @@ <h4>Timing Adjustable</h4>
          
          <p>The user is warned before time expires and given at least 20 seconds to extend the
             time limit with a simple action (for example, "press the space bar"), and the user
-            is allowed to extend the time limit at least ten times; or
+            is given ten opportunities to extend the time limit (with each extension having the
```

Comment:
Probably want to keep the "at least" language in there
```suggestion
            is given at least ten opportunities to extend the time limit (with each extension having the
```